_posts/2015-11-09-foreman-community-newsletter-november.md

```diff
@@ -0,0 +1,99 @@
+the UI and API, which will help in cases where records have been lost 
+due to server rebuilds or other issues. 
+* More detailed host statuses are available, with an extensible 
+mechanism for plugins to be able to add to the green/yellow/red status 
+shown on the host list and dashboard. 
+* Lots of host and host group cloning enhancements and fixes are present 
+in this release, so virtual machine details, Puppet class associations 
+and more are copied correctly, among other improvements. 
+* Creating hosts with host groups, but changing and unsetting certain 
+inherited attributes has become easier with new "inherit" buttons. 
+Similarly, passwords already saved can be removed now. 
+* The smart proxy DNS module has been overhauled, and is now extensible. 
+ New PowerDNS and Route 53 plugins are available already! 
+* Parameter management has seen a few improvements, particularly in the 
+Puppet class edit form.  The host and host group parameters tabs have 
+improved tooltips and layout fixes to improve readability. 
+
+More details in the [announcement](https://groups.google.com/forum/#!topic/foreman-users/lMV2QzaHOuI)
+
+### Foreman 1.9.3 released
+
+Latest release in the 1.9 series is [now available](https://groups.google.com/d/msg/foreman-announce/4jKfFum2RrE/8yjimfEICgAJ) with fixes for some new bugs and regressions.
+
+### New and updated plugins this month
+
+* [foreman_cockpit](https://github.com/theforeman/foreman_cockpit) - first release [announcement](https://groups.google.com/forum/#!topic/foreman-users/QEAQjGs1exQ)
+<br>
+![foreman_cockpit](/static/images/blog_images/2015-11-09-foreman-community-newsletter-november/cockpit.gif)
+* [foreman_salt](https://github.com/theforeman/foreman_salt) and [smart_proxy_salt](https://github.com/theforeman/smart_proxy_salt) - versions 4.0.1 and  2.1.6 respectively. Compatible with Foreman 1.10. [announcement](https://groups.google.com/forum/#!topic/foreman-users/36_Z5oLXBUA)
+* [foreman_discovery](https://github.com/theforeman/foreman_discovery) - version 4.1 (PXE-less discovery via kexec mechanism) [announcement](https://groups.google.com/forum/#!topic/foreman-users/ae29UUEyURc)
+![foreman_cockpit](/static/images/blog_images/2015-11-09-foreman-community-newsletter-november/discovery.gif)
+* [foreman_digitalocean](https://github.com/theforeman/foreman-digitalocean) - version 0.2.1
+* [foreman-xen](https://github.com/theforeman/foreman-xen) - version 0.1.5 and 0.2.1 (ongoing code cleanups and fixes, added support for XenServer pools (only 0.2.x for Foreman 1.10+))
+* [foreman_chef](https://github.com/theforeman/foreman_chef) - version 0.2.0
+* [Ansible integration](https://groups.google.com/forum/#!topic/foreman-dev/cjj2g3bS8ho) - still a work-in-progress
+
+### Upcoming events
+
+* Foreman Meetup London, 20th Nov, 7pm (Clapham Junction) [more details](https://plus.google.com/b/102496134326414788199/events/c7semeinn29f6diq571rnipb7i4)
```

Comment:
Can you make this "Foreman / oVirt Meetup, London ..." as we're doing a joint thing with oVirt.